tests/test_vtk.py

```diff
@@ -39,17 +40,45 @@ def test_validation_pluto(fn):
 @pytest.mark.parametrize(("fn", "geometry"), PLUTO_VTK_FILES)
 def test_parse_pluto_metadata(fn, geometry):
     ds = yt_idefix.load(fn)
-    assert ds._definitions_header == os.path.join(ds.fullpath, "definitions.h")
+    assert os.path.samefile(
+        ds._definitions_header, DATA_DIR.joinpath("pluto_sedov", "definitions.h")
+    )
     assert "unit_length" in ds.parameters
     assert "unit_density" in ds.parameters
     assert "unit_velocity" in ds.parameters
-    assert ds.current_time != -1
-    assert ds.geometry == "cartesian"
-    assert ds.length_unit / ds.velocity_unit == ds.time_unit
-    with pytest.raises(ValueError):
-        ds = yt_idefix.load(fn, definitions_header="definitions2.h")
-    with pytest.warns(UserWarning):
-        ds = yt_idefix.load(fn, definitions_header="definitions2.h", geometry=geometry)
+    assert ds.geometry == geometry
+    assert (
```

Comment:
Calls to `assert_allclose_units` don't need to be chained with plain `assert`. Please simplify.

Another problem here is that this is a parametrised test. Right now we have only one Pluto dataset, so it works, but what you're testing here is specific to this dataset, so it should be done in a separate test